Kang-Minseokk/Invaders-SDP
# Kan 72 change entity skin

## What
This PR changes the game's graphics to fit the team's concept. It includes the following changes:
- Redesigned all spritetypes in DrawManager.
- Changed width and height and changed the graphics to hexadecimal.
- Refactored the loadSprite method to handle correct hexadecimal values ​​and improve exception handling for incorrect data and missing files.

## Why
To align with the concept defined by the team, it was necessary to make precise adjustments to the colors and sizes in the game development.

## How to Test
1. Run the code to check if the graphic is displayed properly.
2. Check if the graphic matches our team concept.
3. Run testLoadSprite() to check if it is tested properly.
